akka-docs/src/main/paradox/dispatchers.md

```diff
@@ -25,6 +25,12 @@ dispatchers in this ActorSystem. If no ExecutionContext is given, it will fallba
 `akka.actor.default-dispatcher.default-executor.fallback`. By default this is a "fork-join-executor", which
 gives excellent performance in most cases.
 
+## Internal dispatcher
+
+To protect the internal Actors that is spawned by the various Akka modules, a separate internal dispatcher is used by default.
+The internal dispatcher can be customized with the setting `akka.actor.internal-dispatcher`.
```

Comment:
Maybe add a line that you can also change dispatchers in a more fine-grained way by customizing the various `dispatcher` config options.